include/messages/topology.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "messages.h"
+#include "util.h"
+
+typedef struct neighborTableEntry_t {
+  struct neighborTableEntry_t *prevNode;
+  struct neighborTableEntry_t *nextNode;
```

Comment:
nit: can be updated to the new code style guides.

Reply:
Agreed on this one, there is a ton of files that use this typedef and that was my hesistation (mainly in bm_protocol). I will make that change, but it will span a lot of bm_protocol files just as a warning!